### ISSUE TITLE
Missing doc for openAL

### ISSUE THREAD

**Issue by mfr-itr** [layeh/gumble#42]
I have some trouble understanding the gumble-openal package. What is it for? From reading the code, it seems to be used to play audio frames into mumble, is that correct?

**Comment by bontibon** [comment]
Yes, it is an audio source and audio sink.

**Comment by mfr-itr** [comment]
Oh, so it can also record the audio stream? Do you have an example/tips about how to do that?

**Comment by bontibon** [comment]
You mean record as in save to disk? If so, that would be better implemented as a custom `AudioListener`.

**Comment by mfr-itr** [comment]
No, in fact I want to send it to liquidsoap : http://savonet.sourceforge.net/doc-svn/reference.html#Source___Input
